workspaces/kiali/.changeset/late-pens-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-kiali-react': minor
+'@backstage-community/plugin-kiali': minor
+---
+
+Fix graph load. Add detail pages
```

Comment:
perhaps expand a bit for the end users what the fix is about, so that we get cleaner changelogs